src/MessagePack.UnityClient/Assets/Scripts/MessagePack/UnicodeUtility.cs

```diff
@@ -0,0 +1,25 @@
+namespace MessagePack.Internal
+{
+    using System;
+    using System.Text;
+    /// <summary>
+    /// unicode manupilation utilities
+    /// </summary>
+    internal static class UnicodeUtility
+    {
+        [ThreadStatic]
+        static char[] Utf8Buffer;
```

Comment:
unused field?

Reply:
yes, this is unused field. I will delete it.